.github/workflows/node-js-ci.yml

```diff
@@ -34,21 +36,14 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - name: Cache node modules
-        uses: actions/cache@v2
-        env:
-          cache-name: cache-node-modules
+      - uses: pnpm/action-setup@v2
         with:
-          path: ~/.npm
-          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('**/package.json') }}
-          restore-keys: |
-            ${{ runner.os }}-build-${{ env.cache-name }}-
-            ${{ runner.os }}-build-
-            ${{ runner.os }}-
+          version: 6.28.0
       - name: Test
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2
         with:
           node-version: ${{ matrix.node-version }}
-      - run: npm ci
-      - run: npm run prettier-check
-      - run: npm test
+          cache: 'pnpm'
+      - run: pnpm install
+      - run: pnpm prettier-check
```

Comment:
Let's move this to its own job. Make another job similar to the `eslint` one above, but have it run `prettier-check` vs. `eslint`.  It should only run on Ubuntu.

Let's rename this from `build` to `test`.

Reply:
Also, could we make `eslint` and `prettier-check` into its own file, like what telescope did?
https://github.com/Seneca-CDOT/telescope/blob/e1e546a58039d7bee642f5ee25fa55865f0a6524/.github/workflows/release.yml#L13-L18

Reply:
That's overkill.  I'd do them as separate jobs here, and they can run in parallel.